src/components/gds-media-card/gds-media-card.stories.js

```diff
@@ -90,4 +90,26 @@ export const Examples = () => html`
       <gds-paragraph size="s">which appears after after the other slots</gds-paragraph>
     </div>
   </div>
+  <div style="margin-bottom: 40px; height: 700px; border: 1px dashed black;">
+    <style>
+      .full-height {
+        --card-height: 100%;
+        --media-card-height: 100%;
+        height: 100%;
```

Comment:
@dominickudiabor I think you should remove this height: 100%; and add that --gds-link-height: 100%;, right? Or is this working correctly on your local machine?

Reply:
I removed it and its still working correctly.

Reply:
What do you mean? :) Screenshot?